src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5399,10 +5399,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>warning action enable</value>
   </data>
   <data name="WRN_ConvertingNullableToNonNullable" xml:space="preserve">
-    <value>Converting null literal or possible null value to non-nullable type.</value>
+    <value>Null literal or possibly null value is being converted to a non-nullable type.</value>
   </data>
   <data name="WRN_ConvertingNullableToNonNullable_Title" xml:space="preserve">
-    <value>Converting null literal or possible null value to non-nullable type.</value>
+    <value>Null literal or possibly null value is being converted to a non-nullable type.</value>
```

Comment:
📝 _If_ a change is made to this line, the trailing `.` should probably be removed for consistency with current messaging guidelines.

Reply:
> memo _If_ a change is made to this line, the trailing `.` should probably be removed for consistency with current messaging guidelines.

There are 778 instances of `.</` in the file.  Fixing this requires a separate commit IMHO.

Reply:
@yecril71pl I think @sharwell's comment applies to warning titles only. Other trailing `.`s would remain.

Reply:
> @yecril71pl I think @sharwell's comment applies to warning titles only. Other trailing `.`s would remain.

I did not perform a global counting but there are more trailing dots than not in messages preceding and following this line.  It still calls for a separate wholesale fix.

Reply:
> It still calls for a separate wholesale fix.

Many of the strings in this file appear in places like build log validation and/or tests for customers downstream. We don't change strings in this file unless we have a need to change. If (and only if) we have a need to change, we construct the new string according to the current guidance at the time of the rewrite.